6.2 Release Notes Cover

sf1919 opened the pull request:
**Description of work.**
The cover for the v6.2 Release Notes.

**To test:**

- remove build/docs/doctrees
- build the documents again and check there are no warnings
- Check the information is correct
- Are the images visible and clearly illustrate the new feature/improvement?
-  Check that the links work
-  Grammar/Spelling check


*There is no associated issue.*

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/ReviewingAPullRequest.html)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there is GUI work does it follow the [GUI standards](http://developer.mantidproject.org/Standards/GUIStandards.html)?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

sf1919 committed:
Summarised v6.2 of Mantid with highlights

sf1919 committed:
Added release notes overall image

martyngigg commented on docs/source/release/v6.2.0/index.rst:
```diff
@@ -4,9 +4,9 @@
 ===========================
 
-.. figure:: ../../images/ImageNotFound.png
+.. figure:: ../../images/ReleaseNotesIndex_v6_2.png
```
This is a nice image. Were we not able to include the workspace calculator in the end?

gvardany replied:
Yes, I think it is worth to include, perhaps at full stretch in the bottom.

sf1919 replied:
I've added the workspace calculator image in but I haven't included it at full stretch as it distorts horribly.

gvardany replied:
Ahh, I meant to stretch the widget in workbench before taking the screenshot, but taking the existing screenshot is fine as well.

martyngigg commented on docs/source/release/v6.2.0/index.rst:
```diff
@@ -16,7 +16,14 @@ Mantid 6.2.0 Release Notes
+This release sees many new algorithms added or improved to extend the functionality of Mantid for users. In addition to this there are some significant new features
+including:-
+
+- Superplot, a new detector widget for the plot window that facilitates over-plotting and the manipulation of overplotted data.
```
Can I ask what we mean by detector widget here? The spectra are related to detectors but they don't have to be...

gvardany replied:
I think originally this was decorator, got mistaken during the clean-up. In any case I think it is a too technical term for the end user, so just saying a new widget is fine. I'd suggest:
`Superplot, a new widget that facilitates over-plotting and data comparison with large number of workspaces.`

sf1919 replied:
Agreed - I misread the description from the workbench notes but I think @gvardany simplified sentence works much better.

martyngigg reviewed (changes requested):
This looks good. Just a couple of small questions.

Also, could we remove the "under construction" note now? I think we're far enough into the process that it's worth doing now.

sf1919 committed:
Minor rewording and image replaced

Under construction notice removed, Superplot description reworded and Workspace Calculator added to image.

martyngigg reviewed (changes requested):
Thanks for making those changes. This is looking good.

I've just got one warning in the build output about a reference that it can't find:
```
﻿﻿/home/mgigg/Code/mantidproject/mantid-nightly/docs/source/release/v6.2.0/indirect_geometry.rst:56: WARNING: undefined label: algm-indirectillenertytransfer (if the link has no caption the label must precede a section header
```
Possibly just a typo with `enerty` -> `energy`?

sf1919 commented:
> 
> 
> Thanks for making those changes. This is looking good.
> 
> I've just got one warning in the build output about a reference that it can't find:
> 
> ```
> ﻿﻿/home/mgigg/Code/mantidproject/mantid-nightly/docs/source/release/v6.2.0/indirect_geometry.rst:56: WARNING: undefined label: algm-indirectillenertytransfer (if the link has no caption the label must precede a section header
> ```
> 
> Possibly just a typo with `enerty` -> `energy`?

To avoid a merge conflict I think this should be updated in #32559 rather than this PR.

martyngigg reviewed (approved):
Yep that makes perfect sense.
